pkg/plugin/vm_backup_item_action.go

```diff
@@ -41,6 +41,12 @@ type VMBackupItemAction struct {
 	log logrus.FieldLogger
 }
 
+const (
+	// AnnIgnoreRestore indicates that the object will be backed up but not restored.
+	// This allows skipping restore-specific checks while ensuring the object is backed up.
+	AnnIgnoreRestore = "cdi.kubevirt.io/velero.ignoreRestore"
```

Comment:
I'm ok with either way but I see a big advantage in being able to explicitly select the VMs that you want to backup without restore by having a vm-specific annotation. The other way seems way more rigid and would force us to skip checks in every VM, but I also like not having to add annotations to the VM. wdyt?

Reply:
I think adding the annotation on every resource will be a pain to the user. Maybe we can support it as a secondary use case but primary should be on the Restore resource itself

Reply:
Okay, that makes sense. I'll change it.

Reply:
I'm wondering if we can come up with a better name for this annotation? Since this is happening at backup time and I don't think `cdi` should be part of the name. Maybe `velero.kubevirt.io/allowPartialBackup`?

Reply:
I was torn between using that name or the current one. I'm happy with the change.

Reply:
yeah after considering this I agree that using `metadataOnly` or `metadataBackup` is better. This brings the possibility of also skipping all consistency-related checks (`canBeSafelyBackedUp` for example) but I think addressing the requested use case is good for the moment.

Reply:
@ShellyKa13 do you have any thoughts on this?

Reply:
After playing with backups I've found there's no way to include annotations into the backup object when using the velero client. This makes it hard to allow resource exclusion when using the client as users would need to patch the backup once it's already started. Should we worry about this?

Reply:
Looks like there is a `--labels` option so maybe make it a label?

Or create a PR in velero repo to add?

Reply:
Do we check any other annotations on the backup resource? If so, should probably make that a label as well (in another pr)

Reply:
Making it a label seems the most straightforward option if we want to do the check in the backup object. I'm not seeing any project that adds annotations to the backup (we don't do it other than in this case), but will ask in their repository if they are ok with having a flag to add them.

Reply:
I have created PR to add a `--annotations` flag in the velero client: https://github.com/vmware-tanzu/velero/pull/8354. Has already 1/2 approval so I think we could go back to the annotation behavior in this PR and expect users to create the backup resource manually until the new flag is released. wdyt?

Reply:
Sure, I'm ok with that